Update hololens-calibration.md (#5327)
MicrosoftDocs/windows-itpro-docs: devices/hololens/hololens-calibration.md

## Calibrating your HoloLens 2

> HoloLens 2 uses eye-tracking technology to improve your experience seeing and interacting with the virtual environment. Calibrating the HoloLens 2 also ensures that it can accurately track your eyes (and the eyes of anyone else who uses the device).  After calibration, holograms will appear correctly even as the visor shifts on your head.

Reviewer: "also" in this context implies that this is a second purpose of calibration. However, no other purpose is mentioned. Recommend removing "also"

Author: There are 2 purposes of ET calibration: one is 'improve your experience seeing and interacting with the virtual environment' and the second one is 'can accurately track your eyes', so I think also needs to stay.

Reviewer: No, the first sentence states that eye tracking is what improves your experience, and calibration improves eye tracking. It's all one thing.

Author: Fair enough. then maybe we need to change the whole paragraph. 
For context: Calibration is needed for both Eye position tracking and Eye Gaze Tracking, Eye position tracking is used to 'improve your experience seeing and interacting with the virtual environment', while eye gaze tracking is used by developers for interaction experiences.

Author: Here is the new proposal: HoloLens 2 uses eye-tracking technology to improve your experience seeing and interacting with the virtual environment. Calibrating the HoloLens 2 ensures comfortable and quality viewing experience and that it can accurately track your eyes (and the eyes of anyone else who uses the device).  After calibration, holograms will appear correctly even as the visor shifts on your head.

Reviewer: OK, if "improved experience" is *one* of the calibration outcomes, I propose turning the two sentences around:

Calibrating the HoloLens 2 ensures that the device can accurately track the eyes and gaze of anyone who uses it. Properly calibrated tracking technology improves your experience of seeing and interacting with the virtual environment. After calibration, holograms will appear correctly even as the visor shifts on your head.

Author: Doing eye tracking calibration to get accurate eye gaze is easily understandable and what most users expect. What is not easy to communicate/understand is that all people need to do eye calibration (for things to appear correctly on the display) even if they are not using any eye tracking (as part of gaze enabled apps). For that reason, I would really like to start with this sentence: 'HoloLens 2 uses eye-tracking technology to improve your experience seeing and interacting with the virtual environment.' We have heard this feedback from users too many times (why do I need to calibrate if I am not using eye tracking?), so I want to make sure this page provides clarity on that.

![Calibration prompt](./images/07-et-adjust-for-your-eyes.png)

> During this process, you'll look at a set of targets (gems). It's fine if you blink, but try to stay focused on the gems during calibration, and not stare at other objects in the room.  This allows HoloLens to learn about your eye position to render your holographic world.

Reviewer: The second sentence no longer works. "During calibration" needs to modify "blinking". "And not stare" is not a parallel structure to the earlier clauses in the sentence (and the sentence has changed from two clauses to three, making i run on). I suggest:

It's fine if you blink during calibration, but try to stay focused on the gems instead of other objects in the room.

> During this process, you'll look at a set of targets (gems). It's fine if you blink, but try to stay focused on the gems during calibration, and not stare at other objects in the room.  This allows HoloLens to learn about your eye position to render your holographic world.

Reviewer 2: (in case you find it easier to merge suggestions than editing the PR content)
```suggestion
During this process, you'll look at a set of targets (gems). It's fine if you blink during calibration, but try to stay focused on the gems instead of other objects in the room.  This allows HoloLens to learn about your eye position to render your holographic world.
```

- Moving objects in your visor out of the way (such as hair)
- Turning on a light in your room or moving out of direct sunlight

If you followed all guidelines and calibration is still failing, you can disable the calibration prompt in Settings. Please also let us know by filing feedback in [Feedback Hub](hololens-feedback.md).

### Disable calibration

You can also disable the calibration prompt by following these steps:

1. Select **Settings** > **System** > **Calibration**.
1. Turn off **When a new person uses this HoloLens, automatically ask to run eye calibration**.

### Calibration data and security

Calibration information is stored locally on the device and is not associated with any account information. There is no record of who has used the device without calibration. This mean new users will get prompted to calibrate visuals when they use the device for the first time, as well as users who opted out of calibration previously or if calibration was unsuccessful.

> Device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be automatically deleted. 

Reviewer 2: The word `Device` seems not to be specific enough to match the rest of the sentence.
My guess is that it should be `The device` (or `Devices`, if the plural was intended).

Author: 'The device' should be used.

> Device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be automatically deleted. 

Reviewer 2: Thank you for your reply confirming which option to use. Here is a suggestion you can merge, unless you want to make the edit yourself in your personal branch.
```suggestion
The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be automatically deleted. 
```
